FAQ.md

```diff
@@ -25,9 +25,11 @@
 
 *   *What do the [labels](https://github.com/swcarpentry/lesson-template/issues?q=is%3Aopen+is%3Aissue) mean?*
 
+    *   `build`: changes to `Makefile` or lesson creation process
```

Comment:
changes to `Makefile`, the validator, or the lesson creation process. (we'll remove `validator` as a separate category)